config/examples/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+namespace: nnf-system
+
+resources:
+- copyoffload-containerprofile.yaml
+- copyoffload-datamovementprofile.yaml
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: nnf-dm-copy-offload
+  newName: ghcr.io/nearnodeflash/nnf-dm-copy-offload
+  newTag: master
```

Comment:
Will this run into issues when k8s has already pulled master? Should we set the pullPolicy to always? I guess this would only affect our systems when running unreleased versions.

Reply:
In your manifest that is made for gitops you should see this replaced with git-version-rev, even for builds out of your workarea.